api_demo/views.py

```diff
@@ -0,0 +1,7 @@
+import json
+from aiohttp import web
+
+
+async def index(request):
+    return web.Response(text=json.dumps({"text": "Hello Aiohttp!"}), status=200)
```

Comment:
Why json?) This is just a string) If you wanna return json object then use `body`  instead of `text`